pybamm/util.py

```diff
@@ -290,52 +287,3 @@ def is_constant_and_can_evaluate(symbol):
             return False
     else:
         return False
-
-
-def install_jax(arguments=None):  # pragma: no cover
```

Comment:
I might have phrased it wrong. We need to add a deprecation warning for this and keep it in the codebase for at least a year.